include/tvm/tir/schedule/schedule.h

```diff
@@ -242,6 +242,17 @@ class ScheduleNode : public runtime::Object {
   /******** Schedule: loop binding/annotation ********/
   /******** Schedule: cache read/write ********/
   /******** Schedule: reduction ********/
+  /*!
+   * \brief Factor a reduction block by the specified loop
+   * \details See python/tvm/tir/schedule/schedule.py
```

Comment:
Hmmmm I am not sure, but perhaps it is better to have one complete doc on C++ side, another on python side, so probably you might want to copy from python's doc

Reply:
I don't know how to present the code examples from Python side here. Do you think it's acceptable to omit the code examples?

Reply:
We probably dont need the code examples, because it is python-specific

Reply:
Got it. Will do that tomorrow.